### DISCUSSION TITLE
Unable to download

### DISCUSSION THREAD

**Discussion by Im937** [aclap-dev/video-downloadhelper#975] [Help and Q&A]
### Your question

I cannot load the video from streaming website (http://ch3plus.com/) 
on Chrome latest version [126.0.6478.127](http://126.0.6478.127/) (official) (64 bit)

2days ago and today 07.05.2024 08.51 IST I downloaded a clip from that website perfectly fine 
![download failedVersion 9 0 2 12 (stable)](https://github.com/aclap-dev/video-downloadhelper/assets/174785835/f6c270ac-56f6-48d9-adf9-6c6cec375ed6)
but a few minutes later it showed error msg.

I thought it was about license limitations , so I paid for the licences.
However, it keeps showing error msg.

Download failed
Sadly we failed at downloading that specific media. We try to support as many websites as possible, so it would help a lot if you could report that error (it's anonymous!).
![download failedVersion 9 0 2 12 (stable)](https://github.com/aclap-dev/video-downloadhelper/assets/174785835/613e9ca2-0a4b-43b3-967a-e353289adff3)

Could you pls advise any solutions?

### Addon & CoApp version and details

Version [9.0.2.12](http://9.0.2.12/) (stable), target google, locale th.
Companion App installed ([v2.0.19)](http://v2.0.19)/) Recheck
C:\Program Files\DownloadHelper CoApp\[vdhcoapp.exe](http://vdhcoapp.exe/)

**Comment by GrampaWildWilly** [comment]
I do not work for VDH.  I'm just another user.

I'm running Windows 11 Pro 64-bit version 23H2 build 22631.3810, Firefox 127.0.2 64-bit, licensed VDH 9.0.2.12a1 beta, CoApp 2.0.19.

I visited your web site & picked something at random.  VDH recognized 2 things.  The first thing was of duration 1:02.  VDH successfully downloaded it & VLC played it.  It looked like an advertisement for some soap opera kind of thing with lots of girls & guys crying, looking deeply into each other's eyes, laughing, the usual shlock.

The other thing VDH offered said it was 40 seconds long.  But after it spent over 4 minutes downloading & not telling me what percentage was complete, I stopped the download.  What I had played fine.  It appeared to be a livestream of a news broadcast.  It played just fine.

Of course, I have no idea what anybody was saying in either of these items.  I'm guessing the language they were speaking is Thai, but that's a guess.  It's no guess that I didn't understand it, whatever language it was.

So there you have it.  It works fine for me.  I suggest you get the VDH beta for Chrome, since you're using Chrome.  Maybe that will make a difference.  You might also consider switching to Firefox.  There are some reports in here of things that don't work on Chrome that work fine on Firefox.  I know.  The browser any individual uses is like a religion.  I know I use Firefox & will not even consider Chrome, Edge, Opera, Tor, Waterfox, or anything else.  I've used Firefox since it was called Netscape.  I'm not switching.  So if you don't want to switch, I wouldn't be surprised.  But it is true that there's a bunch of things that our users have encountered that work great on Firefox & fail miserably on Chrome.  The decision is yours.

**Reply by Im937** [reply]
Thank you for your reply. I may try to switch to Firefox. 
Now I can download after finding one of your answer about those antivirus programs. Uninstall and clear internal storage and then ping! Everything works smoothly as usual.

Thank you again for your contribution.
